{{ cookiecutter.repo_name }}/.flake8

```diff
@@ -1,3 +1,5 @@
+# As of now, flake8 does not natively support configuration via pyproject.toml
```

Comment:
some might wonder why we have configs in `.flake8` as a separate file instead of having them under `pyproject.toml`